### DISCUSSION TITLE
next/image svg

### DISCUSSION THREAD

**Discussion by erwanvivien** [vercel/next.js#27101] [Ideas]
### Describe the feature you'd like to request

When importing `.png` for example we get a `blurDataURL:` param in the object.

Could we have the same for SVG ?

### Describe the solution you'd like

Adding a blurData for SVGs

### Describe alternatives you've considered

I considered exporting images from my SVGs, but it's a bit tedious

**Comment by fschwalm** [comment]
Any updates or workarounds?

thank you!
